--- meson-gxm-q200.dts	2021-03-14 21:18:56.844349100 +0800
+++ meson-gxm-q200-mod.dts	2021-03-13 04:55:45.030947600 +0800
@@ -73,9 +73,27 @@
 	};
 };
 
+/*
 &sd_emmc_a {
 	brcmf: wifi@1 {
 		reg = <1>;
 		compatible = "brcm,bcm4329-fmac";
 	};
 };
+*/
+
+/* This is connected to the Bluetooth module: */
+&uart_A {
+	status = "okay";
+	pinctrl-0 = <&uart_a_pins>, <&uart_a_cts_rts_pins>;
+	pinctrl-names = "default";
+	uart-has-rtscts;
+
+	bluetooth {
+		compatible = "qcom,qca9377-bt";
+		enable-gpios = <&gpio GPIOX_17 GPIO_ACTIVE_HIGH>;
+		max-speed = <2000000>;
+		clocks = <&wifi32k>;
+		clock-names = "lpo";
+	};
+};
\ No newline at end of file
